tests/test_utils.py

```diff
@@ -196,7 +197,8 @@ def test_cleanup(self):
             self.assertFalse(os.path.exists('%s/.pr-title.tmp' % tempdir))
 
     def test_resolve_dep_oe(self):
-        """Test resolve dependency with Open Embedded"""
-        # Note(allenh1): we're not going to test the hard-coded resolutions.
-        self.assertEqual(resolve_dep('tinyxml2', 'oe'), 'libtinyxml2')
-        self.assertEqual(resolve_dep('p2os_msgs', 'oe'), 'p2os-msgs')
+        """Test resolve dependency with OpenEmbedded"""
+        self.assertEqual(resolve_dep('tinyxml2', 'openembedded')[0],
+            ['libtinyxml2@meta-oe'])
+        with self.assertRaises(UnresolvedDependency):
+            resolved_deps = resolve_dep('p2os_msgs', 'openembedded')
```

Comment:
It might be good to leave a note here about why this should raise an assert so that people (i.e. me) don't forget.

Reply:
:-) Done!